### ISSUE TITLE
This audit prove that truecrypt was very secure. 

### ISSUE THREAD

**Issue by LifeIsStrange** [veracrypt/VeraCrypt#33]
https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Publications/Studies/Truecrypt/Truecrypt.html
There is some bugs discover so you should fix them :)

**Comment by idrassi** [comment]
I don't understand how we can call a software very secure when there are proven and practical kernel privilege escalation vulnerabilities, plus a dozen buffer overruns and overflows, and of course a weak key derivation.

If we follow this logic, Microsoft doesn't have to issue urgent updates for kernel privilege escalation discovered in Windows...after all, this doesn't affect security!!

Concerning this latest audit, I have already said [on twitter](https://twitter.com/VeraCrypt_IDRIX/with_replies) that it doesn't bring anything new with regards to what it is already known, either on VeraCrypt side or the other public audits and reports.

For example, the /dev/random issue they "discovered" was known since 2011 and it has been documented by Ubuntu Privacy Remix Team on their report on August 4th 2011: https://www.privacy-cd.org/downloads/truecrypt_7.0a-analysis-en.pdf

As for the static code analysis, I used many of them for VeraCrypt back in 2014 and I was able to find many issues and fixed them.

Can someone tell me what I missed in this BSI report? I don't see anything new...
